database/mongodb/README.md

```diff
@@ -1,68 +1,98 @@
+ beginning in the `%environment` section.
 
-Another important feature to notice is that you can also refer to these environment variables on runtime with the prefix `$SINGULARITYENV_` , following this example `$SINGULARITYENV_HOSTNAME` and `$SINGULARITYENV_PORT` can be referenced from runtime.
+Another important feature to notice is that you can also refer to these
+ environment variables on runtime with the prefix `$SINGULARITYENV_` , following
+ this example `$SINGULARITYENV_HOSTNAME` and `$SINGULARITYENV_PORT` can be
+ referenced from runtime.
 
 To run the definition file and build our container, we will need to:
 
-```
-$ sudo singularity build mongodb.sif mongodb.def
+```sh
+sudo singularity build mongodb.sif mongodb.def
 ```
 
-This will generate a `mongodb.sif` Singularity image. From there, you will need to bind the `/data/db` folder inside the container to a folder in the host (In this example we will bind it to a new folder in the host called `mongodb/db` ). This folder is where the data from the database such as logs and/or journals are stored. Open a command line and then just run:
+This will generate a `mongodb.sif` Singularity image. From there, you will need
+ to bind the `/data/db` folder inside the container to a folder in the host
+ (In this example we will bind it to a new folder in the host called
+ `mongodb/db` ). This folder is where the data from the database such as logs
+ and/or journals are stored. Open a command line and then just run:
 
-```
-$ mkdir -p mongodb/db
-$ export SINGULARITY_BINDPATH="mongodb/db:/data/db"
-$ singularity shell mongodb.sif
+```sh
+mkdir -p data/db
+export SINGULARITY_BINDPATH="data/db:/data/db"
+singularity shell mongodb.sif
 ```
 
-Once we exported the `SINGULARITY_BINDPATH` environment variable, the `mongodb/db` folder on the host binds to the `/data/db` folder from the container.
+Once we exported the `SINGULARITY_BINDPATH` environment variable, the
+`mongodb/db` folder on the host binds to the `/data/db` folder from the
+container.
 
-After this we are ready to run an instance, for this we run the following command:
+After this we are ready to run an instance, for this we run the following
+command:
 
-```
-$ sudo singularity instance start mongodb.sif mongo
+```sh
+singularity instance start mongodb.sif mongo
 ```
 
-We can see there the `mongo` server running and the shell of the `mongo` server waiting for input. From here you can start setting up your database by, for example, adding a new user.
+We can see there the `mongo` server running and the shell of the `mongo` server
+ waiting for input. From here you can start setting up your database by, for
+ example, adding a new user.
 
 To add a new user in the database and give it a specific role, you can do so by:
 
-First, shell into the container and run:
+First, shell into the instance and run:
 
-```
-$ sudo singularity shell mongodb.sif
+```sh
+sudo singularity shell instance://mongodb
```

Comment:
Why was the prompt identifier removed?

Reply:
is marked a markdown lint when after that you don't show the output